### DISCUSSION TITLE
Non answering forum!

### DISCUSSION THREAD

**Discussion by oferardi** [AxisCommunications/discussions#261] [ACAP]
Hi

Why only certain discussions are answered?

 Were is axis team? Lost in the desert?

**Comment by xtrdustin** [comment]
I'm sure they have a lot on their plate, but it does make it hard to get things working with their APIs (in my case, ONVIF) if I don't know if my questions will ever be answered. I've seen one of the devs commenting about questions not having enough information. Hopefully they can let us know what information is needed for a given ticket if it doesn't have everything they need. I'm happy to put in the leg work on my end, if it will help them answer my questions, I just need to know what I missed.

**Comment by mattias-kindborg-at-work** [comment]
Hi @oferardi and @xtrdustin!

Thank you for being active on GitHub Discussions. As of today we've revised the [Axis support guidelines](https://github.com/orgs/AxisCommunications/discussions/13), and any new question will have an automatic reply, providing transparency and attempting to align expectations. 

This forum should be considered as a complement to already existing Axis support channels, with the benefit of being open. Unfortunately we have to prioritize questions based on the estimated effort required to find an answer. Well crafted questions, with elaborate context and reproducible steps, will get more attention and will have a higher chance of getting answered.

I hope that you understand, and thank you for the time you spend answering questions here in the forum!

**Reply by xtrdustin** [reply]
@mattias-kindborg-at-work Hi Mattias! Thanks for taking the time to address this. I understand that answering questions here is probably just one of many things you're responsible for within Axis, and there are only so many hours to go around. The Axis helpdesk can be very helpful in some cases, but there's a limit to what they can answer in terms of questions about the API (and they've said that development questions should be asked here, not there). So it's it makes my job easier when I can get answers to more development-oriented questions here. Hopefully we can make your job easier by asking better questions. 

But if for some reason we're not able to get an answer here, are there any other options for getting support for development/API issues? Should we try the helpdesk again at that point?

**Reply by mattias-kindborg-at-work** [reply]
Hi @xtrdustin and thanks for the feedback!

I sympathize with anyone stuck on a problem that seems impossible to solve, while the deadline is getting closer and your manager is requesting progress updates... It's the worst place to be in.

The best way to get support from Axis depends on the problem at hand. We've got a few repositories here in the [GitHub organization](https://github.com/AxisCommunications) that are tailored towards different domains of information. E.g. the [ACAP Computer Vision SDK examples](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples) has multiple ACAP app examples, each showcasing something for developers building containerized CV applications. If you have a question relating any of these examples, the repository itself would be the place to start. It has issues and it has discussions, both available to all developers. The repositories are monitored by code owners, and often you'll get an answer to your question.

We have GitHub Discussions on the organizational level, this is where we are now. This discussion forum is relevant for questions that doesn't relate to a specific repository, but it will also get less attention from code owners.

Finally we have the Axis Helpdesk. The helpdesk definitely accept technical questions, but it is not available to all developers. It is not open and publicly accessible as GitHub Discussions is. If you have access to the helpdesk, you ought to start there.
